src/style.css

```diff
@@ -2,0 +1,69 @@
+body {
+  margin: 0 8px;
```

Comment:
According to the checklist, it's recommended to be consistent with your margins (Add only top or bottom, but not both) to avoid potential margin collapse. Here you are adding margin to both sides of the body, which is not consistent with the checklist guidelines.